### DISCUSSION TITLE
How make union of events?

### DISCUSSION THREAD

**Discussion by feonit** [zerobias/effector#359] [Q&A] [answered]
If ```merge``` api gives us event that triggered if one of set fire, may be need api like if all events of set will be fired?
analogy with Promise.race() [merge]  and Promise.all() [union]

**Comment by zerobias** [comment, marked as answer]
You can use [combineEvents](https://github.com/sergeysova/patronum/tree/master/src/combine-events) from `patronum`

![image](https://user-images.githubusercontent.com/15912112/85196490-38e3e080-b2e3-11ea-9aaf-2768e4acab58.png)
